main.php

```diff
@@ -1,13 +1,19 @@
 <?php
 
 function ExecuteSQL ($query) {
-	return "SQL result\n";
+	return sprintf("Some result from \"%s\"\n\n", $query);
 }
 
 function Main() {
-	$userNamesSQL = "SELECT name FROM Users";
-	$userNames = ExecuteSQL($userNamesSQL);
+	$getUserNamesSQL = "SELECT UserName FROM Users;";
+	$userNames = ExecuteSQL($getUserNamesSQL);
 	print $userNames;
+
+	print("Please enter your username: ");
+	$userName = rtrim(fgets(STDIN), "\n");
+	$getUserDataSQL = "SELECT * FROM Users WHERE UserName = " . $userName . ";";
```

Comment:
SQL query strings shouldn't be concatenated, these are susceptible to SQL injection.